### ISSUE TITLE
Stack and the call for contribution this issue

### ISSUE THREAD

**Issue by lwm** [haskellweekly/haskellweekly.github.io#138]
Thanks for all your work on Haskell Weekly - it's great.

I wanted to note that in the 'Call for Contribution' section this latest issue, the Stack related issue was closed. That's fine, of course, things are moving along but it got me thinking. Might we get a 'Stack needs people to help all the time, forever' note in there to try and get people involved? If that is a bit much, perhaps linking out to the 'newcomer-friendly' issues, which might also be better - https://github.com/commercialhaskell/stack/issues?q=is%3Aopen+is%3Aissue+label%3A%22awaiting+pull+request%22+label%3a%22newcomer+friendly%22. Up to you!

**Comment by tfausak** [comment]
Thanks for the heads up! Turns out we were linking to the wrong issue. I fixed that problem in 39ac790a82bfc50fdc7fe493d0c502da4f6bb37d.

To the larger suggestion of linking to Stack's issues that are friendly to newcomers: I'm hesitant to put any project there every time. There are lots of Haskell projects that always need people to help, so why should Stack be singled out? That being said, we can certainly try to include a Stack issue every week, and having them labeled like that makes our job easier 😄 

/cc @alexeyzab
